src/ProjectNeon/Assets/State/Templates/BattleState.cs

```diff
@@ -7,35 +7,59 @@ public class BattleState : ScriptableObject
     [SerializeField] private EnemyArea enemies;
     
     // @todo #1:10min Replace this with Member once implemented.
-    [SerializeField] private Enemy tempTarget;
+    [SerializeField] private Member tempTarget;
 
-    public List<Member> GetAllies()
+    public Member[] GetAllies()
     {
         /**
          * @todo #55:30min We need references to Member Decks in BattleState so we can return all Members with their 
          *  correct state on GetAllies and GetEnemies methods. After we get these references, update GetAllies and 
          *  GetEnemies methods to return the correct Members
          */
-        List<Member> allies = new List<Member>();
-        allies.Add(new Member().Init(party.characterOne));
-        allies.Add(new Member().Init(party.characterTwo));
-        allies.Add(new Member().Init(party.characterThree));
-        return allies;
+        return new []
+        {
+            new Member().Init(party.characterOne), 
+            new Member().Init(party.characterTwo), 
+            new Member().Init(party.characterThree)
+        };
     }
 
-    public List<Member> GetEnemies()
+    public Member[] GetEnemies()
     {
-        List<Member> enemyList = new List<Member>();
-        enemies.enemies.ForEach(
-            enemy =>
-            {
-                Character character = new Character();
-                character.Bust = enemy.image;
-                character.Stats = enemy.stats;
-                character.ClassName = enemy.enemyName;
-                enemyList.Add(new Member().Init(character));
-            }
-        );
-        return enemyList;
+        return enemies.Enemies.Select(x => new Member()).ToArray();
     }
+
```

Comment:
@SilasReinagel I'm not sure about these targeting methods: it seems fair that `BattleState` should know which targets are available or not for each team, because I imagine that `BattleState` will possess the data about both groups. On the other hand, this implementation seems to need a polymorphic solution; we have three methods that do the same basic thing: get a target for someone.

Also, it does not seems possible to play some card 'from one player character to another' (i.e, 'Neo casts Heal on Trinity')  because we don't have `self` parameter in `GetPossiblePlayerTargets`, making this impossible to forecast how each card will baheve depending on which player character we chose to play. Another example would be attacks based on player and enemy position, like a laser beam that cuts through all enemies in a single line or some grenade with blast area damage: available targets would change depending the player character we chose.

I think that we could try to make just one method to select possible targets since their behavior is too similar